docs/content/administration/config-cheat-sheet.en-us.md

```diff
@@ -761,6 +763,21 @@ and
 - `SEND_BUFFER_LEN`: **100**: Buffer length of mailing queue. **DEPRECATED** use `LENGTH` in `[queue.mailer]`
 - `SEND_AS_PLAIN_TEXT`: **false**: Send mails only in plain text, without HTML alternative.
 
+## Override Email Headers (`mailer.override_header`)
+
+:::warning
+This is empty by default, use it only if you know what you need it for.
+:::
+
+examples would be:
+
+```ini
+[mailer.override_header]
+Reply-To = test@example.com, test2@example.com
+Content-Type = text/html; charset=utf-8
```

Comment:
I guess we shouldn't provide `Replay-To` and `Content-Type` as examples ....

Reply:
@wxiaoguang do you have other examples then?

Reply:
TBH, no idea because IMO few real users would really need such feature. Maybe it could be left as undocumented until it gets enough interests.

Reply:
we should not have undocumented features!!!

we could make docu as minimal as possible, witch I think I did here